### PR TITLE
Impl. new specs

### PR DESCRIPTION
**Pull request by Fyzu**
Related to https://github.com/rispa-io/rispa-core/issues/20

### CODE REVIEW

File: bin/build.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-const { build } = require('@rispa/core/events')
-const core = require('@rispa/core')
+const { init } = require('@rispa/core')
 
-core(build())
+init(context => {
```
Comment (Laiff) [inline bin/build.js]: А зачем тут анонимная функция а не из api метод?

Comment (Fyzu) [reply bin/build.js]: Забыл заменить, исправлю

**Comment by Fyzu** [comment]
[132. Implement plugin validator and rework plugins](https://trello.com/c/kie0n7Wg/132-132-implement-plugin-validator-and-rework-plugins)

**Comment by Fyzu** [comment]
[132. Implement plugin validator and rework plugins](https://trello.com/c/kie0n7Wg/132-132-implement-plugin-validator-and-rework-plugins)

**Review by Laiff — APPROVED** [review]
Related to https://github.com/rispa-io/rispa-core/issues/20

**Pull request merged**